Application/MicroBooter/MicroBooter.cpp

```diff
@@ -3,7 +3,7 @@
 //  <No description>
 //
 //  Microsoft dotNetMF Project
-//  Copyright �2004 Microsoft Corporation
+//  Copyright ©2004 Microsoft Corporation

```

Comment:
Ugh! :face_palm: Sigh... Thanks for finding this. Unfortunately fixing the encoding on mass is likely to cause more problems than it will help. Given that the current plan on [vNext](https://github.com/NETMF/netmf-interpreter/tree/v5_dev) is to start with a clean slate so we can also do code style clean ups as we essentially import code into a new branch we can define and normalize the encoding for each file type. (And ideally find a way to detect deviations before the get into the repo)